lib/qb_integration/invoice.rb

```diff
@@ -0,0 +1,34 @@
+module QBIntegration
+  class Invoice < Base
+    attr_accessor :flowlink_invoice
+
+    def initialize(message = {}, config)
+      super
+      @flowlink_invoice = payload[:invoice]
+    end
+
+    def create
+      if qb_invoice = invoice_service.find_by_invoice_number
```

Comment:
You're not doing anything with the "qb_invoice" variable. Is there any reason why that "if" couldn't just be "if invoice_service.find_by_invoice_number" ?